### PR TITLE
Story#567

### PR DESCRIPTION
**Pull request by rejneesh1**

### CODE REVIEW

File: src/components/date-type/date-type.html [hunk 1]
```diff
@@ -1,48 +1,97 @@
...
+      (ionCancel)="canceled()"
+      [(ngModel)]="data.value"
+      (ngModelChange)="checkForValidation()"
+    ></ion-datetime>
 
-      <ion-datetime [disabled]="data?.autoCapture" displayFormat="{{data?.dateFormat}}" *ngIf="!data?.validation.min && !data?.validation.max"
-       (ionCancel)="canceled()" [(ngModel)]="data.value" [max]="futureDate"
-      (ngModelChange)="checkForValidation()"></ion-datetime>
+    <ion-datetime
+      [disabled]="data?.autoCapture || (data.isEditable!=undefined && !data.isEditable)"
```
Comment (kiranharidas187) [inline src/components/date-type/date-type.html:43]: this condition u need to apply for all ion-datetime tags in this page

File: src/components/input-type/input-type.html [hunk 2]
```diff
@@ -10,22 +9,43 @@ <h4 [ngClass]="{'_validQuestion': data?.value}" class="flexbox" >
...
+      [inputIndex]="inputIndex"
+      [data]="data"
+      [enableQuestionReadOut]="enableQuestionReadOut"
+    ></question-heading>
+  </h4>
 </ion-col>
-<div class="_tip _flex-box "> {{data?.tip}}</div>
+<div class="_tip _flex-box">{{data?.tip}}</div>
 <div margin-top>
+  {{data.isEditable|json}}
```
Comment (kiranharidas187) [inline src/components/input-type/input-type.html]: remove this

**Review by kiranharidas187 — COMMENTED** [review]
Please resolve this

File: src/components/date-type/date-type.html (hunk 1 above)
Comment (rejneesh1) [reply src/components/date-type/date-type.html:43]: its in 4 places in this page . for all total 4 ion-datetime tags

File: src/components/input-type/input-type.html (hunk 2 above)
Comment (rejneesh1) [reply src/components/input-type/input-type.html]: removed

**Pull request merged**